cmd/server/flags.go

```diff
@@ -525,4 +525,22 @@ var flags = []cli.Flag{
 		Hidden:  true,
 		// TODO(485) temporary workaround to not hit api rate limits
 	},
+	//
+	// secrets encryption in DB
+	//
+	&cli.StringFlag{
+		EnvVars: []string{"WOODPECKER_ENCRYPTION_KEY"},
```

Comment:
it should be possible to set both secret as direct env or `WOODPECKER_ENCRYPTION_KEY_PATH` that would read it from file so that docker secrets can be used for this